src/main/java/fi/otavanopisto/kuntaapi/server/system/SystemUtils.java

```diff
@@ -0,0 +1,21 @@
+package fi.otavanopisto.kuntaapi.server.system;
+
+/**
+ * 
+ * System utilities
+ * 
+ * @author Heikki Kurhinen
+ */
+public class SystemUtils {
+  
+  private SystemUtils(){};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)